Mackbellemore/soen-390-team07
# feat(server): Add log middleware to insert AFTER post/patch

#

## Changes this PR introduces
Had to do a 180 since logdna's free tier doesnt let us make parallel api requests.

- Added a logger middleware that executes after POST / PATCH requests
- If the logging fails for whatever reason the request is still sent as a success (we have logdna for that in case)
- For now basic action ex: "Created a bike" with a date and email is stored



## How to test this PR

- Make some POST requests everywhere (Besides user) and check GET /logs to see if your email was logged with the right action







## Notes for reviewers

- a note



## Checklist
